applications/services/loader/loader_cli.c

```diff
@@ -60,6 +64,20 @@ static void loader_cli_open(FuriString* args, Loader* loader) {
     furi_string_free(app_name);
 }
 
+static void loader_cli_close(Loader* loader) {
+    FuriString* app_name = furi_string_alloc();
+
+    if(!loader_get_application_name(loader, app_name)) {
+        printf("No application is running\r\n");
+    } else if(!loader_signal(loader, FuriSignalExit, NULL)) {
+        printf("Application \"%s\" has to be closed manually\r\n", furi_string_get_cstr(app_name));
+    } else {
+        printf("Applicaton \"%s\" was closed\r\n", furi_string_get_cstr(app_name));
```

Comment:
```suggestion
        printf("Application \"%s\" was closed\r\n", furi_string_get_cstr(app_name));
```

Reply:
Thanks, I should really use a spell checker.